docs/using-gitbase/functions.md

```diff
@@ -19,6 +19,17 @@ To make some common tasks easier for the user, there are some functions to inter
 |`uast_children(blob) blob`| returns a flattened array of the children UAST nodes from each one of the UAST nodes in the given array.              |
 |`loc(path, blob) json`| returns a JSON map, containing the lines of code of a file, separated in three categories: Code, Blank and Comment lines. |
 |`version() text`| returns the gitbase version in the following format `8.0.11-{GITBASE_VERSION}` for compatibility with MySQL versioning. |
+||||||| merged common ancestors
+|`is_remote(reference_name)bool`| check if the given reference name is from a remote one                                                          |
+|`is_tag(reference_name)bool`| check if the given reference name is a tag                                                                         |
```

Comment:
We should change "check" by "checks" to be consistent with the rest of the descriptions.

Reply:
Changed.